src/global.scss

```diff
@@ -4,6 +4,30 @@ div::-webkit-scrollbar {
   display: none;
 }
 
+.mainDiv::-webkit-scrollbar {
+  display: block;
+  width: 8px;
+}
+
+.mainDiv::-webkit-scrollbar-thumb {
+  background-color: rgba(140, 140, 140, 0.5);
+  border-radius: 4px;
+}
+
+.mainDiv {
```

Comment:
You can move this CSS back into the JSX file. Another reason for doing this is that the app layout is accessed less frequently compared to the global.scss, which mainly contains classes that are used across multiple elements all over the app.